### PR TITLE
added nocom crash :D

### PR DESCRIPTION
**Pull request by TaxEvasiqn**

### CODE REVIEW

File: src/main/java/widecat/meteorcrashaddon/modules/NoComCrash.java [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+import net.minecraft.util.Hand;
+import net.minecraft.util.hit.BlockHitResult;
+import net.minecraft.util.math.BlockPos;
+import net.minecraft.util.math.Direction;
+import net.minecraft.util.math.Vec3d;
+import java.util.Objects;
+import java.util.Random;
+
+public class NoComCrash extends Module{
+    final Random      r       = new Random();
```
Comment (SIMULATAN) [inline src/main/java/widecat/meteorcrashaddon/modules/NoComCrash.java:19]: Tf

File: src/main/java/widecat/meteorcrashaddon/modules/NoComCrash.java [hunk 2]
```diff
@@ -0,0 +1,55 @@
...
+    int i = 0;
```
Comment (SIMULATAN) [inline src/main/java/widecat/meteorcrashaddon/modules/NoComCrash.java:20]: What's this for

File: src/main/java/widecat/meteorcrashaddon/modules/NoComCrash.java [hunk 3]
```diff
@@ -0,0 +1,55 @@
...
+    Vec3d pickRandomPos() {
+        int x = this.r.nextInt(0xFFFFFF);
+        int y = 255;
+        int z = this.r.nextInt(0xFFFFFF);
+        return new Vec3d(x, y, z);
+    }
+    private final SettingGroup sgGeneral = settings.getDefaultGroup();
+    private final Setting<Integer> amount = sgGeneral.add(new IntSetting.Builder()
```
Comment (SIMULATAN) [inline src/main/java/widecat/meteorcrashaddon/modules/NoComCrash.java:28]: AFAIK you can just use the add method with no need to get the default group

File: src/main/java/widecat/meteorcrashaddon/modules/NoComCrash.java [hunk 4]
```diff
@@ -0,0 +1,55 @@
...
+        .description("How many packets to send to the server per tick.")
+        .defaultValue(15)
+        .min(1)
+        .sliderMax(100)
+        .build()
+    );
+
+    private final Setting<Boolean> autoDisable = sgGeneral.add(new BoolSetting.Builder()
+        .name("auto-disable")
+        .description("Disables module on kick.")
```
Comment (SIMULATAN) [inline src/main/java/widecat/meteorcrashaddon/modules/NoComCrash.java:39]: I'd rather write "Disable the module when disconnected" or smth

**Review by SIMULATAN — CHANGES_REQUESTED** [review]
Found some minor things

File: src/main/java/widecat/meteorcrashaddon/modules/NoComCrash.java (hunk 3 above)
Comment (C10udburst) [reply src/main/java/widecat/meteorcrashaddon/modules/NoComCrash.java:28]: uh I'm not sure, i have not seen this being done differently. i would leave that as it is

File: src/main/java/widecat/meteorcrashaddon/modules/NoComCrash.java (hunk 4 above)
Comment (C10udburst) [reply src/main/java/widecat/meteorcrashaddon/modules/NoComCrash.java:39]: i would remove this setting entirely because it is not being used at all

**Comment by C10udburst** [comment]
looks mostly fine. could you remove the unused variables and fix the formatting slightly (remove the 14 spaces and add a space before `{`)?

**Comment by TaxEvasiqn** [comment]
i have fixed my idiocy

File: src/main/java/widecat/meteorcrashaddon/modules/NoComCrash.java (hunk 4 above)
Comment (SIMULATAN) [reply src/main/java/widecat/meteorcrashaddon/modules/NoComCrash.java]: :O didn't even notice that LOL

**Pull request merged**